### PR TITLE
Update to Stryker 1.0

### PR DESCRIPTION
**Pull request by sbergen**

### CODE REVIEW

**Comment by sbergen** [comment]
Waiting for https://github.com/stryker-mutator/stryker-net/pull/1759 to be reviewed before continuing with this work.

File: Tests/Responsible.Tests.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <TargetFramework>net5.0</TargetFramework>
```
Comment (rouke-broersma) [inline Tests/Responsible.Tests.csproj:4]: fyi this should not be a requirement to be able to use stryker v1 :)

Comment (sbergen) [reply Tests/Responsible.Tests.csproj:4]: Thanks for pointing it out! However, ReproducibleBuilds requires .NET 5.0.300 SDK or newer, so targeting .NET 5 in my **tests** simplifies the GH Actions runner set up.

Comment (rouke-broersma) [reply Tests/Responsible.Tests.csproj:4]: Mhm I had not realized Reproducible Builds required such a new SDK. Weird.

**Pull request merged**